inference_rules.adoc

```diff
@@ -228,8 +228,17 @@ time.
 
 ==== Constraints for the Closed division
 
-There are two benchmark suites, one for Datacenter systems and one for Edge (defined herein as non-datacenter) systems. A Datacenter submission must use ECC in their DRAM and HBM memories, and ECC must be enabled for all performance and accuracy runs. No requirements are imposed on SRAM. The suites share multiple benchmarks, but characterize them with different requirements. Read the specifications carefully.
+There are two benchmark suites, one for Datacenter systems and one for Edge (defined herein as non-datacenter) systems. The minimum requirements for a datacenter system are defined below:
 
+===== Minimum requirements of a datacenter system
+====== ECC
+A Datacenter submission must use ECC in their DRAM and HBM memories, and ECC must be enabled for all performance and accuracy runs. No requirements are imposed on SRAM.
+
+====== Networking
+A datacenter system must be equipped with all the necessary networking required by the system architecture described in the <<LoadGen Operation>> section.  The details of the networking components must be described in the appropriate field of the System JSON (https://github.com/mlcommons/policies/blob/master/submission_rules.adoc).  All necessary networking must be populated if power is measured along with performance. 
```

Comment:
Does populating the networking and actually using them cause a difference in power usage?

Reply:
Yes, we'll get the idle power usage and for network interface it may not vary significantly based on load. But it does not look good to have this requirement unless loadgen actually serves the queries over the network interface for the server scenario. Can we have this modification for loadgen server scenario and then this requirement will be trivially satisfied by all submitters?

Reply:
As I mentioned in the other comment thread - while we would like transparency into the populated components during a power submission in order to make power submissions more reviewable and transparent, we discussed in the Power WG that this was not the way to do it.  We will instead use the System JSON and file a different PR to achieve the same goal but without having to enforce bandwidth requirements or change the loadgen operation rules as this PR does.

Reply:
Regardless of the variation  of load or not , we submit system power measured during the performance phase, so the power from components like NICs will add to measured power.   The ask really is that you document your system components in the system Json and not about having a mandatory number of NICs .  What part of the description above does not look good. Could you copy/paste the exact part ?

Reply:
Sure. The problem is with the "necessary" part. 

all the **necessary** networking required by the system architecture described 

When server scenario does not mandate network usage submitters are open to not use network at all. If **necessary** means only the components which are actually being used for the submission run (like for interacting with the power server),  the rule is fine. But if **necessary** in turn means all the networking components which are required to server the inference load to the server, that is a much bigger requirement.

Reply:
"A datacenter system must be equipped with all the necessary networking required by the system architecture described in the <<LoadGen Operation>> section."

@arjunsuresh  - adding the full sentence for clarity

As @anirban-ghosh  wrote, this will be discussed separately in Inference WG and through a separate PR , not tied for Power WG or power submissions.   What a data center system definition entails is a separate topic.

Reply:
Thank you @s-idgunji for the clarification. I understand that what the `necessary` part is part of a separate discussion. But consider the below line: 

> All necessary networking must be populated if power is measured along with performance.

Here, the statement makes sense only if we can measure the performance without using networking. Can we avoid the confusion by renaming the section heading to **IO** and changing the description as follows? 

> A datacenter system must be equipped with all the necessary networking or IO devices as required by the system architecture described in the LoadGen Operation section. The details of the networking components must be described in the appropriate field of the System JSON (https://github.com/mlcommons/policies/blob/master/submission_rules.adoc). 

**The above change prohibits the usage of Network mounted disks**